File: fix_index.js
const fs = require('fs');
const content = fs.readFileSync('index.html', 'utf8');
const startIdx = content.indexOf('<!-- Hidden PDF Template -->');
if (startIdx !== -1) {
    const newContent = content.substring(0, startIdx) + '</body>\n</html>';
    fs.writeFileSync('index.html', newContent, 'utf8');
    console.log("Fixed index.html properly with UTF-8");
} else {
    console.log("Template not found in index.html");
}

File: fix_index.py
import sys

with open("index.html", "r", encoding="utf-8") as f:
    content = f.read()
    
# Remove the old #pdf-wrapper block
start_idx = content.find('<!-- Hidden PDF Template -->')
if start_idx != -1:
    content = content[:start_idx] + '</body>\n</html>'

with open("index.html", "w", encoding="utf-8") as f:
    f.write(content)
    
print("Fixed index.html properly with UTF-8")
